.eslintrc.js

```diff
@@ -47,6 +51,53 @@ module.exports = {
   rules: {
     'no-restricted-imports': [2, restrictedModules],
     'no-restricted-modules': [2, restrictedModules],
+    '@kbn/eslint/no-restricted-paths': [
+      'error',
+      {
+        zones: [
+          {
+            // when tslint is removed we will check *.ts files as well
+            target: './src/legacy/.*js$',
```

Comment:
I'm not opposed to limiting to .js files, but do you think it's necessary? ESLint will only process the files .js files, and this will automatically apply to .ts files as soon as we get them in here without this limitation.

Reply:
didn't want to force https://github.com/elastic/kibana/pull/33826 to comment/fix all problem places in `*.ts` files.

Reply:
We're going to limit all existing rules to .js files in that PR, so I wouldn't worry about it.